Content.Server/Chat/Commands/SuicideCommand.cs

```diff
@@ -82,6 +83,13 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 return;
             }
 
+            //Checks to see if the CannotSuicide tag exits, ghosts instead.
+            if(EntitySystem.Get<TagSystem>().HasTag(owner, "CannotSuicide"))
+            {
+                EntitySystem.Get<GameTicker>().OnGhostAttempt(mind, false);
```

Comment:
This should give an error if ghosting isn't possible, ala the ghost command.

Reply:
Done